models/model_runner.py

```diff
@@ -65,13 +65,22 @@ def execute_reid_action():
         script_args.extend(optional_args)
         call(script_args)
 
-    if args.action == RE_ID_EVAL:
+    elif args.action == RE_ID_EVAL:
         script_args = ['/home/bar_cohen/miniconda3/envs/mmtrack/bin/python', './fast-reid/tools/train_net.py',
                        '--config-file', args.reid_config, '--eval-only', 'MODEL.DEVICE', 'cuda:0', 'DATASETS.DATASET',
                        args.dataset]
         script_args.extend(optional_args)
         call(script_args)
 
+    elif args.action == RE_ID_VIZ:
+        # python3 demo/visualize_result.py --config-file ./configs/DukeMTMC/bagtricks_R101-ibn.yml --dataset-name DukeMTMC  --output results/DukeMTMC-1.8.21/ --vis-label --label-sort descending --opts MODEL.WEIGHTS checkpoints/duke_bot_R101-ibn.
+
+        script_args = ['/home/bar_cohen/miniconda3/envs/mmtrack/bin/python', './fast-reid/demo/visualize_result.py',
+                       '--config-file', args.config, '--eval-only' '--label-sort', 'descending', 'MODEL.DEVICE', 'cuda:0', 'DATASETS.DATASET',
+                       args.dataset, '--opts', 'MODEL.WEIGHTS', 'checkpoints/duke_bot_R101-ibn' ]
+        script_args.extend(optional_args)
```

Comment:
I think we don't need the optional_args here. I added it in the other two cases since there we might want to run the model without any checkpoint. However, here I don't see why we would want to visualize results we get without any checkpointl